docs/pages/setup/guides.mdx

```diff
@@ -4,17 +4,17 @@ description: Teleport Installation and Configuration Guides.
 layout: tocless-doc
 ---
 
-<TileSet>
-  <Tile icon="wrench" title="Terraform Provider" href="./guides/terraform-provider.mdx">
-    How to configure Teleport Cloud, Open Source, and Enterprise with Terraform Provider.
-  </Tile>
-  <Tile icon="wrench" title="Docker" href="./guides/docker.mdx">
-    Getting started with Teleport Open Source using Docker.
-  </Tile>
-  <Tile icon="wrench" title="Fluentd Event Forwarder" href="./guides/fluentd.mdx">
-    Forwarding events with Fluentd and Teleport Events Handler.
-  </Tile>
-  <Tile icon="wrench" title="EC2 tags as Teleport Nodes" href="./guides/ec2-tags.mdx">
-    How to setup Teleport node labels based on EC2 tags.
-  </Tile>
-</TileSet>
+<ul>
```

Comment:
Q: Are we going away from the tile set layout, or do you just think that list works better for these guides?

Reply:
It works really well for a small number of items, but for large amount of articles lists are the best